.github/workflows/backend-testing.yml

```diff
@@ -22,10 +22,11 @@ jobs:
     steps:
       - name: Checkout Debezium UI
         uses: actions/checkout@v2
-      - name: Set up JDK 11
+      - name: Set up JDK 17
         uses: actions/setup-java@v1
         with:
-          java-version: 11
+          distribution: 'temurin'
```

Comment:
why `temurin`? Don't we prefer vanilla `adopt`/`adopt-hotspot`?

Reply:
It's the same used in the core repository.